Spp 002 nl (#157)
inbo/protocolsource: source/spp/mnm/spp_002_nl_locatie-eval_mnm/02_onderwerp.Rmd

# Onderwerp

## Definities en afkortingen

```{=html}
<!--
Definities van begrippen en afkortingen die noodzakelijk zijn om dit document te begrijpen en het protocol op een gepaste manier te kunnen uitvoeren.
-->
```

> **Celgekoppelde type**: celgebaseerde habitat(sub)type of regionaal belangrijk biotoop waarbij de GRTS-rastercel wordt toegewezen op basis van ruimtelijke overlap van het habitat(sub)type of regionaal belangrijke biotoop met de rastercel, maar het type treft niet noodzakelijk het celmidden van de rastercel (in tegenstelling tot een celmiddengekoppelde type)

Reviewer: ```suggestion
**Celgekoppelde type**: celgebaseerde habitat(sub)type of regionaal belangrijk biotoop waarbij een GRTS-rastercel wordt toegewezen op basis van ruimtelijke overlap van het habitat(sub)type of regionaal belangrijke biotoop met de rastercel, maar het type treft niet noodzakelijk het celmidden van de rastercel (in tegenstelling tot een celmiddengekoppelde type)
```

> **Celgekoppelde type**: celgebaseerde habitat(sub)type of regionaal belangrijk biotoop waarbij de GRTS-rastercel wordt toegewezen op basis van ruimtelijke overlap van het habitat(sub)type of regionaal belangrijke biotoop met de rastercel, maar het type treft niet noodzakelijk het celmidden van de rastercel (in tegenstelling tot een celmiddengekoppelde type)

Reviewer: ```suggestion
**Celgekoppelde type**: celgebaseerde habitat(sub)type of regionaal belangrijk biotoop waarbij de GRTS-rastercel wordt toegewezen indien ze ruimtelijk overlapt met het habitat(sub)type of regionaal belangrijke biotoop, maar het type treft niet noodzakelijk het celmidden van de rastercel (in tegenstelling tot een celmiddengekoppelde type)
```

> **Celgekoppelde type**: celgebaseerde habitat(sub)type of regionaal belangrijk biotoop waarbij de GRTS-rastercel wordt toegewezen op basis van ruimtelijke overlap van het habitat(sub)type of regionaal belangrijke biotoop met de rastercel, maar het type treft niet noodzakelijk het celmidden van de rastercel (in tegenstelling tot een celmiddengekoppelde type)

Reviewer: ```suggestion
**Celgekoppelde type**: celgebaseerde habitat(sub)type of regionaal belangrijk biotoop waarbij de GRTS-rastercel wordt toegewezen op basis van ruimtelijke overlap van het habitat(sub)type of regionaal belangrijke biotoop met de rastercel. Het type treft hierbij niet noodzakelijk het celmidden van de rastercel (in tegenstelling tot een celmiddengekoppelde type)
```

**Celkartering**: het ruimtelijk in kaart brengen van het voorkomen van het doeltype binnen een rastercel van 32 m op 32 m.

> Dit resulteert in een polygoon die uitsluitend uit het doeltype bestaat (en gelegen is binnen de grenzen van de rastercel), ook celkarteringspolygoon genoemd.

Reviewer: ```suggestion
Dit resulteert in een polygoon binnen de grenzen van de rastercel die uitsluitend uit het doeltype bevat, ook celkarteringspolygoon genoemd.
```

> **Celmiddengekoppelde type**: celgebaseerde habitat(sub)type of regionaal belangrijk biotoop waarbij het midden van een GRTS-rastercel het type treft

Reviewer: ```suggestion
**Celmiddengekoppelde type**: celgebaseerde habitat(sub)type of regionaal belangrijk biotoop waarbij een GRTS-rastercel wordt toegewezen indien het midden van de GRTS-rastercel het type treft
```

> **Doeltype**: het habitat(sub)type of regionaal belangrijke biotoop (rbb) waarvoor de steekproeflocatie is getrokken voor een specifiek meetnet.

Reviewer: Suggestion to add a brief definition of "doelpopulatie"; either append to this definition or as a separate definition in this section (or later in the text, where the term is used, so far without explanation). 
Something along the lines:

"Het geheel van types dat beschouwd moet worden om op niveau Vlaanderen uitspraken te doen over een milieudruk vormt de doelpopulatie van het bij die milieudruk horende meetnet (zie MNM)."

> Er is geen overlap voor tussen sub- en hoofdtypes omdat afzonderlijke steekproeflocaties worden getrokken voor hoofd- en subtypes.

Reviewer: ```suggestion
Er is geen overlap tussen sub- en hoofdtypes omdat afzonderlijke steekproeflocaties worden getrokken voor hoofd- en subtypes.
```

> Er is geen overlap voor tussen sub- en hoofdtypes omdat afzonderlijke steekproeflocaties worden getrokken voor hoofd- en subtypes.

Reviewer: I would probably leave out the note on separate sampling locations, as it seems to cause confusion. The overlap concerns overlap between main and subtypes (which then implies (absence of) geographical overlap, although there can still be overlap for cell-coupled types, right?).

So, to clarify, I would instead write something like: 

"Er is geen overlap (...) in de zin dat ofwel een hoofdtype ofwel de subtypes van een hoofdtype het doeltype of de doeltypes voorstellen."

(Note that in stratified sampling, strata do per definition not overlap.)

**rbb**: regionaal belangrijk biotoop

**steekproefeenheid**: een individueel element van de steekproef van een specifiek meetnet met attribuutwaarden verbonden aan het design (vb. GRTS-rangnummer, doeltype, panelnummer).
Het ruimtelijke aspect is de **meetlocatie** en is in dit protocol een rastercel van 32 m op 32 m, omdat deze cellen de basis (de zogenaamde *sample support*) zijn van terrestrische types in MNM (met uitzondering van 7220).

**vervangeenheid**: een lokale vervanging van een steekproefeenheid, wanneer de rastercel van de getrokken steekproefeenheid het doeltype niet bevat (al dan niet in het celmidden), en in dit protocol voor celgebaseerde types ook wel **vervangcel** genoemd.

## Doelstelling en toepassingsgebied

```{=html}
<!--
Doelstelling: omschrijving in woorden waarvoor het protocol dient.
Voorbeelden wat er met de gegevens kan gedaan worden.
Probeer de doelstelling zo veel mogelijk generiek te beschrijven en vermijd projectspecifieke doelstellingen.
Toepassingsgebied: bespreek hier onder welke condities het protocol kan gebruikt worden.
-->
```

### Doelstelling

> Dit protocol beschrijft hoe terrestrische steekproefeenheden ("meetlocaties") worden beoordeeld op de aanwezigheid van het doeltype in een meetnet van de Meetnetten Natuurlijk Milieu (MNM).

Reviewer: ```suggestion
Dit protocol beschrijft hoe terrestrische steekproefeenheden (meetlocaties) worden beoordeeld op de aanwezigheid van het doeltype in een meetnet van de Meetnetten Natuurlijk Milieu (MNM).
```
term has been introduced in the definition section, so ok to use without quotation marks, I guess

De steekproefeenheden zijn het resultaat van een GRTS-trekking voor een specifiek meetnet van de MNM.

> Ze worden getrokken uit het meetnetspecifieke steekproefkader, welke de best beschikbare oplijsting vormt van de eenheden die tot de doelpopulatie van het meetnet behoren.

Reviewer: suggestion to define "doelpopulatie" (see comment above)

Het steekproefkader van een meetnet is een beperking van het basissteekproefkader van alle meetnetten van de MNM op basis van de lijst types die de doelpopulatie van elk meetnet definiëren.

> Per meetnet kunnen er extra ruimtelijke restricties gelden binnen types van het basissteekproefkader.

Reviewer: Suggestion to adapt along the lines: 

"Daarnaast kunnen er extra ruimtellijke restricties gelden binnen een meetnetspecifieke steekproefkader (vb. niet alle standplaatsen van een type zijn noodzakelijk relevant voor de milieudruk in kwestie)."

(The "meetnetspecifieke steekproefkader" has already been defined in the foregoing sentence as "een beperking (...)", so seems ok to start from that defintion here.)

> Het basissteekproefkader is aangemaakt op basis van originele datalagen en datalagen die zijn afgeleid van de [Habitatkaart](https://www.vlaanderen.be/inbo/natura-2000-monitoring/bwk-en-habitatkaart/) of de Watervlakkenlaag, en houdt rekening met de evaluatie van meetlocaties in het kader van het meetnet biotische habitatkwaliteit en reeds uitgevoerde locatie-evaluaties in het kader van de MNM.

Reviewer: ```suggestion
Het basissteekproefkader is aangemaakt op basis van originele datalagen en datalagen die zijn afgeleid van de [Habitatkaart](https://www.vlaanderen.be/inbo/natura-2000-monitoring/bwk-en-habitatkaart/) of de Watervlakkenlaag, en houdt rekening met reeds uitgevoerde locatie-evaluaties in het kader van de MNM en het Meetnet Biotische Habitatkwaliteit.
```

De steekproefeenheid van terrestrische habitats is een GRTS-rastercel van 32 m op 32 m waar het doeltype volgens de Habitatkaart voortkomt.
Een polygoon behorend tot een habitat of rbb kan dus meerdere GRTS-rastercellen omvatten.
Een onderscheid wordt gemaakt tussen celmidden- en celgekoppelde types.
Voor de meeste terrestrische types gebeurt de koppeling van GRTS‐rastercellen aan polygonen via het celmidden: de GRTS‐rastercel krijgt een habitat- of rbb-type toegekend van een polygoon als het centrum van de rastercel ermee overlapt.

> De koppeling van GRTS‐rastercellen aan habitatpolygonen gebeurt evenwel celgebaseerd voor types die vooral voorkomen als kleine of smalle patches of in consequent lage aandelen in het polygoon (nl. de zogenaamde phab‐waarden in de Habitatkaart).

Reviewer: re "evenwel celgebaseerd":

I am wondering whether this can be misleading, so suggestion to reformulate to not contradict the fact that both cell center- and cell-coupled types are cell-based.

Hierbij krijgt de GRTS‐rastercel een habitattype toegekend van een polygoon als het ermee overlapt.
De celgekoppelde types zijn habitat(sub)types 1320, 2150, 6210_sk, 6430_bz en 7150 en de rbb‐types brem‐ en gaspeldoornstruweel (rbbsg) en doornstruweel (rbbsp).

De getrokken steekproefeenheden dienen het doeltype te bevatten waarvoor de eenheid is getrokken, welke is afgeleid van de meest recente Habitatkaart.
Het doeltype dient voor te komen ofwel in het celmidden bij celmiddengekoppelde types of ergens in de cel bij celgekoppelde types.
Toch kan het zijn dat in de praktijk de actueel aanwezige vegetatie in (het celmidden van) de rastercel ter plaatse afwijkt en niet behoort tot het doeltype waarvoor de meetlocatie is getrokken.

> Dit kan komen door een fouten of onvolkomenheden in de het (basis)steekproefkader (bronlagen), een verandering op het terrein sinds het vorige terreinbezoek of het voorkomen van een complex in de habitatkaart (waarbij meerdere types binnen één habitatpolygoon voorkomen).

Reviewer: ```suggestion
Dit kan komen door fouten of onvolkomenheden in het (basis)steekproefkader (bronlagen), een verandering op het terrein sinds het vorige terreinbezoek of het voorkomen van een complex in de habitatkaart (waarbij meerdere types binnen één habitatpolygoon voorkomen).
```

> In het steekproefkader van celmiddengekoppelde types wordt een correctie toegepast op basis van het aandeel habitat in de polygoon (de phab-waarde).

Reviewer: ```suggestion
For celmiddengekoppelde types wordt in het steekproefkader een correctie toegepast op basis van het aandeel habitat in de polygoon (de phab-waarde).
```

Reviewer: also something like:

"in het in de Habitatkaar angegeven aandeel"?

> Die correctie houdt in dat polygonen (of stukken daarvan) met een lagere phab proportioneel minder worden opgenomen in het steekproefkader.

Reviewer: Possibly append:

"(...) dan polygonen met hogere phab-waarde."

To link to larger probability statement in next sentence?

Hierdoor vergroot de kans dat de terreinmedewerker een positieve locatie-evaluatie kan uitvoeren tijdens het bezoek aan een gebied.

> Echter, waar zulke polygonen dan resteren en getrokken worden, moeten ze wel een geldige eenheid opleveren (als ze die bevatten), anders genereer je bias door ondervertegenwoordiging van populatie-eenheden in polygonen met lagere phab.

Reviewer: ```suggestion
Echter, waar zulke polygonen dan resteren en getrokken worden, moeten ze wel een geldige steekproefeenheid opleveren (als ze die bevatten), anders genereer je bias door ondervertegenwoordiging van populatie-eenheden in polygonen met lagere phab.
```

Reviewer: consider to specify: "potential bias";
whether there will be bias (in the sense of bias of an estimator) depends on whether missing units differ systematically from non-missing units in the outcome variable of interest.

Voor celmiddengekoppelde types is bijgevolg een lokale vervanging vereist.

De aanwezigheid van het doeltype in de rastercel dient dus ter plaatse te worden gecontroleerd.

> Indien het doeltype er niet voorkomt, dient de terreinmedewerker via de procedure in het vervangprotocol binnen een vooraf bepaald gebied een andere steekproefeenheid te selecteren die wel geschikt is: de vervangeenheid.

Reviewer: link to the subprotocol or refer to section where it is specified?

Om de voor de hand liggende fouten efficiënt op te sporen, wordt de steekproefeenheid en het geheel van de vervangeenheden voor het terreinbezoek geëvalueerd op basis van een desktopscreening.

Als de steekproefeenheid of vervangeenheid geschikt is, dient in de geschikte rastercel een kartering te worden uitgevoerd van het doeltype, verderop de celkartering genoemd.
Met deze informatie zijn veldmedewerkers zonder of met minder vegetatiekundige expertise in staat de bemonstering of installatie van meettoestellen op een vegetatiekundig geschikte locatie uit te voeren.

> Dit kan met tussenkomst zijn van een extra randomisatie in de geschikte deelzone van de vervangeenheid.

Reviewer: explain or refer to explanation of "extra randomisatie"

> Dit protocol omschrijft de werkwijze van de desktopscreening en de evaluatie van de aanwezigheid van het doeltype in terrestrische celgebaseerde steekproefeenheden van de MNM, en daarmee verbonden het lokaal vervangen van niet-geschikte steekproefeenheden en de celkartering van geschikte terrestrische steekproefeenheden.

Reviewer: ```suggestion
Dit protocol omschrijft de werkwijze van de desktopscreening en de evaluatie ter plaatse van de aanwezigheid van het doeltype in terrestrische celgebaseerde steekproefeenheden van de MNM, en daarmee verbonden het lokaal vervangen van niet-geschikte steekproefeenheden en de celkartering van geschikte terrestrische steekproefeenheden.
```

### Toepassingsgebied

> Dit protocol is van toepassing voor de evaluatie, vervanging en celkartering van terrestrische eenheden van GRTS-steekproeven getrokken voor abiotische monitoring in een meetnet van de MNM, met uitzondering van het habitattype 7220.

Reviewer: Overlap with previous paragraph - intended?

### Beperkingen van de procedure

Het veldwerk in dit protocol kan enkel uitgevoerd worden:

> - binnen de (sub)optimale periode voor kartering en is afhankelijk van het doeltype (zie [Vriens et al. (2011)](https://purews.inbo.be/ws/portalfiles/portal/1904260/Vriens_etal_2011_BiologischeWaarderingskaart.pdf)).

Reviewer: ```suggestion
- binnen de (sub)optimale periode voor kartering die afhankelijk is van het doeltype (zie [Vriens et al. (2011)](https://purews.inbo.be/ws/portalfiles/portal/1904260/Vriens_etal_2011_BiologischeWaarderingskaart.pdf)).
```

Reviewer: alternative suggestion (depending on what is meant):
```suggestion
- binnen de (sub)optimale periode voor kartering 
- afhankelijk van het doeltype (zie [Vriens et al. (2011)](https://purews.inbo.be/ws/portalfiles/portal/1904260/Vriens_etal_2011_BiologischeWaarderingskaart.pdf)).
```

# Onderwerp

## Definities en afkortingen

```{=html}
<!--
Definities van begrippen en afkortingen die noodzakelijk zijn om dit document te begrijpen en het protocol op een gepaste manier te kunnen uitvoeren.
-->
```

> **Celgekoppelde type**: celgebaseerde habitat(sub)type of regionaal belangrijk biotoop waarbij een GRTS-rastercel het type bevat

Reviewer 2: verwarrende definitie, alternatief: GRTS rastercelwaaraan een habitat(sub)type of rbb is toegekend obv ruimtelijke overlap

Reviewer 3: Eventueel aan te passen naar: "**Celgebaseerd type**: Type waarvoor de GRTS rastercel wordt toegewezen op basis van ruimtelijke overlap met een habitat(sub)type of rbb met de rastercel in plaats van het celmidden, zie ook **Celmiddengekoppelde type**"

> **Celkartering**: het ruimtelijk in kaart brengen van het voorkomen van het doeltype binnen een rastercel van 32 m op 32 m, welke resulteert in de celkarteringpolygoon

Reviewer 2: celkarteringpolygoon?? wat is dat dan?

Reviewer 3: Vervang door: 
`**Celkartering**:  het ruimtelijk in kaart brengen van het voorkomen van het doeltype binnen een rastercel van 32 m op 32 m. Dit resulteert in een polygoon die uitsluitend uit het doeltype bestaat, ook celkarteringspolygoon genoemd.`

**Celmiddengekoppelde type**: celgebaseerde habitat(sub)type of regionaal belangrijk biotoop waarbij het midden van een GRTS-rastercel het type treft

**Doeltype**: het habitat(sub)type of regionaal belangrijke biotoop (rbb) waarvoor de steekproeflocatie is getrokken voor een specifiek meetnet.

> Er komt geen overlap voor tussen sub- en hoofdtypes.

Reviewer 2: Alt: er worden afzonderlijke steekproeflocaties getrokken voor hoofd-en subtypes (geen overlap).

Reviewer 3: Akkoord, al snap ik de achterliggende techniciteit niet.

> De habitat(sub)types en regionaal belangrijke biotopen komen voor terrestrische eenheden in de MNM overeen met de strata.

Reviewer 2: begrijp deze zin niet echt

Author: Waarsch te technisch voor protocol - te deleten

**GRTS-steekproef**: ruimtelijk gebalanceerde willekeurige steekproef gebaseerd op het GRTS algoritme (Stevens & Olsen, 2004).
Een GRTS-steekproef is gebaseerd op een grid van GRTS-rastercellen met een bepaalde dimensie.

> Elke GRTS-rastercel krijgt een uniek GRTS-rangnummer op basis van het omgedraaide base4-adres.

Reviewer 2: base4-adres???

Een steekproef met grootte n wordt bekomen door de GRTS-rastercellen met de n laagste rangnummers te selecteren binnen de doelpopulatie.
In de MNM wordt een GRTS-steekproef getrokken voor elk meetnet afzonderlijk.

**MNM**: Meetnetten Natuurlijk Milieu.
Deze meetnetten hebben als doel de toestand en trend van milieudrukken in het natuurlijk milieu op het niveau van Vlaanderen op te volgen ter ondersteuning van het natuurbeleid.
Hiervoor dient de abiotiek in Natura 2000-habitats en regionaal belangrijke biotopen te worden gemonitord.
Eén meetnet volgt één milieudruk op in een specifiek compartiment (grondwater, oppervlaktewater, bodem, inundatiewater of atmosfeer).

**rbb**: regionaal belangrijk biotoop

> **steekproefeenheid**: een individueel element van de steekproef van een specifiek meetnet met design-attributen.

Reviewer 2: design-attributen??

Het ruimtelijke aspect is de **meetlocatie** en is in toepassing van dit protocol, namelijk voor terrestrische types met een cel als sample support (celgebaseerd), een rastercel van 32 m op 32 m.

> 

Reviewer 2: sample support?? lijkt me idem als sample size en dan begrijp ik de zin niet

Author: sample support is de dimensie, geometrie, vorm ed geassocieerde met de meting, in dit geval dus de cel. ik snap de mogelijke verwarring -> ik verduidelijk.

> **vervangeenheid**: een lokale vervanging van een steekproefeenheid, en in dit protocol voor celgebaseerde types ook wel **vervangcel** genoemd.

Reviewer 2: vermelden wanneer een vervanging aan de orde is?

Reviewer 3: Ok.
Voorstel:
**vervangeenheid**: een lokale vervanging van een steekproefeenheid wanneer de getrokken GRTS-rastercel niet voldoet aan de voorwaarden, en in dit protocol voor celgebaseerde types ook wel **vervangcel** genoemd.

## Doelstelling en toepassingsgebied

```{=html}
<!--
Doelstelling: omschrijving in woorden waarvoor het protocol dient.
Voorbeelden wat er met de gegevens kan gedaan worden.
Probeer de doelstelling zo veel mogelijk generiek te beschrijven en vermijd projectspecifieke doelstellingen.
Toepassingsgebied: bespreek hier onder welke condities het protocol kan gebruikt worden.
-->
```

### Doelstelling

Dit protocol beschrijft hoe terrestrische steekproefeenheden ("meetlocaties") worden beoordeeld op de aanwezigheid van het doeltype in een meetnet van de Meetnetten Natuurlijk Milieu (MNM).
De steekproefeenheden zijn het resultaat van een GRTS-trekking voor een specifiek meetnet van de MNM.
Ze worden getrokken uit het meetnetspecifieke steekproefkader, welke de best beschikbare oplijsting vormt van de eenheden die tot de doelpopulatie van het meetnet behoren.
Het steekproefkader van een meetnet is een beperking van het basissteekproefkader van alle meetnetten van de MNM op basis van de lijst types die de doelpopulatie van elk meetnet definiëren.
Per meetnet kunnen er extra ruimtelijke restricties gelden binnen types van het basissteekproefkader.

> Het basissteekproefkader is aangemaakt op basis van originele datalagen en datalagen die zijn afgeleid van de Habitatkaart of de Watervlakkenlaag, en bevatten bestaande locatie-evaluaties.

Reviewer 2: locatie-evaluties??
Habitatkaart en Watervlakkenlaag referenties of linken toevoegen

Reviewer 3: Ik neem aan dat er gedoeld wordt op bestaande, bemeten, locaties van MHQ en andere meetnetten waar het habitattype op puntniveau is bepaald?
Voorstel:
`en bevatten goedgekeurde, bemeten meetpunten van andere meetnetten zoals het meetnet habitatkwaliteit`

Author: 'en rekening houdend met eerdere nog geldige locatie-evaluaties'

Author: verduidelijkt

Een polygoon behorend tot een habitat of rbb kan dus meerdere GRTS-rastercellen omvatten.
Een onderscheid wordt gemaakt tussen celmidden- en celgekoppelde types.
Voor de meeste terrestrische types gebeurt de koppeling van GRTS‐rastercellen aan polygonen via het celmidden: de GRTS‐rastercel krijgt een habitat- of rbb-type toegekend van een polygoon als het centrum van de rastercel ermee overlapt.
De koppeling van GRTS‐rastercellen aan habitatpolygonen gebeurt evenwel celgebaseerd voor types die vooral voorkomen als kleine of smalle patches of in consequent lage aandelen in het polygoon (nl. de zogenaamde phab‐waarden in de Habitatkaart).
Hierbij krijgt de GRTS‐rastercel een habitattype toegekend van een polygoon als het ermee overlapt.
De celgekoppelde types zijn habitat(sub)types 1320, 2150, 6210_sk, 6430_bz en 7150 en de rbb‐types brem‐ en gaspeldoornstruweel (rbbsg) en doornstruweel (rbbsp).

De getrokken steekproefeenheden dienen het doeltype te bevatten waarvoor de eenheid is getrokken, welke is afgeleid van de meest recente Habitatkaart.
Het doeltype dient voor te komen ofwel in het celmidden bij celmiddengekoppelde types of ergens in de cel bij celgekoppelde types.

> Toch kan het zijn dat in de praktijk de actueel aanwezige vegetatie in (het celmidden van) de rastercel ter plaatse afwijkt en niet behoort tot het doeltype waarvoor de meetlocatie is getrokken.

Reviewer 2: wegens fouten in het (basis)steekproefkader

Reviewer 3: Alternatief:
`Dit kan komen door een fout in de bronlagen, een verandering op het terrein sinds het vorige terreinbezoek of het voorkomen van een complex in de habitatkaart. Waarbij meerdere types binnen één habitatpolygoon voorkomen.`

Author: toegevoegd

In het steekproefkader van celmiddengekoppelde types wordt een correctie toegepast op basis van het aandeel habitat in de polygoon (de phab-waarde).

> Die correctie houdt in dat polygonen (of stukken daarvan volgens level3-cellen) met een lagere phab proportioneel minder worden opgenomen in het steekproefkader.

Reviewer 2: level 3 cellen???

Author: te complex - deleten

> Hierdoor vergroot de kans dat de terreinmedewerker een bemonstering kan uitvoeren tijdens het bezoek aan een gebied.

Reviewer 3: bemonstering > positieve locatie-evaluatie

> Echter, waar zulke polygoonen dan resteren en getrokken worden, moeten ze wel een geldige eenheid opleveren (als ze die bevatten), anders genereer je bias.

Reviewer 2: polygonen ipv polygoonen

Reviewer 2: ik zou die bias voor de duidelijkheid dan specifiek omschrijven. Ik interpreteer dat dus habitats en rbbs met typisch lage phab waarden ondervertegenwoordigd dreigen te zijn in de steekproef, correct?

Author: ja, toe te voegen: 'door ondervertegenwoordiging van populatie-eenheden in polygonen met lagere phab'

Voor celmiddengekoppelde types is bijgevolg een lokale vervanging vereist.

De aanwezigheid van het doeltype in de rastercel dient dus ter plaatse te worden gecontroleerd.
Indien het doeltype er niet voorkomt, dient de terreinmedewerker via de procedure in het vervangprotocol binnen een vooraf bepaald gebied een andere steekproefeenheid te selecteren die wel geschikt is: de vervangeenheid.
Om de voor de hand liggende fouten efficiënt op te sporen, wordt de steekproefeenheid en het geheel van de vervangeenheden voor het terreinbezoek geëvalueerd op basis van een desktopscreening.

Als de steekproefeenheid of vervangeenheid geschikt is, dient in de geschikte rastercel een kartering te worden uitgevoerd van het doeltype, verderop de celkartering genoemd.
Met deze informatie zijn veldmedewerkers zonder of met minder vegetatiekundige expertise in staat de bemonstering of installatie van meettoestellen op een vegetatiekundig geschikte locatie uit te voeren.

> Dit kan met tussenkomst zijn van een extra randomisatie in de geschikte zone.

Reviewer 2: ```suggestion
Dit kan met tussenkomst zijn van een extra randomisatie in de geschikte deelzone van de vervangeenheid.
```

Dit protocol omschrijft de werkwijze van de desktopscreening en de evaluatie van de aanwezigheid van het doeltype in terrestrische celgebaseerde steekproefeenheden van de MNM, en daarmee verbonden het lokaal vervangen van niet-geschikte steekproefeenheden en de celkartering van geschikte terrestrische steekproefeenheden.

### Toepassingsgebied

Dit protocol is van toepassing voor de evaluatie, vervanging en celkartering van terrestrische eenheden van GRTS-steekproeven getrokken voor abiotische monitoring in een meetnet van de MNM, met uitzondering van het habitattype 7220.

> ### Beperkingen van de procedure

Reviewer 2: is kennis van de vegetatiekundige definitie en de herkenning daarvan op terrein ook geen vereiste (alvast voor het gedeelte celkartering en evaluatie?)

Reviewer 3: Deels akkoord. Staat ook wel hieronder onder vereiste competenties. Suggestie om eventueel hieronder toe te voegen:
`- door een persoon met voldoende vegetatiekundige kennis die de Natura 2000 habitattypes op het terrein kan onderscheiden.`

Het veldwerk in dit protocol kan enkel uitgevoerd worden:

> -   binnen de (sub)optimale periode voor kartering en is afhankelijk van het doeltype (zie)

Reviewer 2: (zie) ???

Author: Vriens L., Bosch H., De Knijf G., De Saeger S., Oosterlynck P., Guelinckx R., T’jollyn F., Van Hove M. & Paelinckx D.
(2011). De Biologische Waarderingskaart. Biotopen en hun verspreiding in Vlaanderen en het Brussels
Hoofdstedelijk Gewest. Mededelingen van het Instituut voor Natuur‐ en Bosonderzoek, INBO.M.2011.1. Instituut voor Natuur‐ en Bosonderzoek, Brussel.
